main.cpp

```diff
@@ -20,67 +20,96 @@ You are going to write 10 UDTs in project3.
-    
-Sub Object 3: Environment Controls
-    Name 3 things you'll find on the:    Environment Controls
-        1) climate control
-        2) Hazard button 
-        3) air vent
-    Name 3 things you can do with the:   Environment Controls
-        1) turn on/off air conditioning
-        2) turn on/off hazard lights
-        3) change air vent speed
-
-Sub Object 4: Infotainment System
-    Name 3 things you'll find on the:    Infotainment System
-        1) Multi-standard radio
-        2) Smartphone pairing
-        3) compass directions
-    Name 3 things you can do with the:   Infotainment System
-        1) turn on/off radio
-        2) choose between satellite or AM/FM radio
-        3) set the driver or passenger air temperature in fahrenheit 
-
-Sub Object 5: Seat 
-    Name 3 things you'll find on the:    Seat
-        1) seat cover 
-        2) headrest
-        3) rocker switch for adjustment
-    Name 2 things you can do with the:   Seat
-        1) moving the seat forwards and backwards with rocker switch
-        2) ajust headrest height 
+1)  Fill in #1 - 4 with a random UDT in plain english
+    These 4 UDTs do not need to be related.
+        a) For each plain-english UDT, write out 5 traits or properties and 3 things it can do
+        b) remember: these properties will eventually become primitives
 */
+
+/*
+ example:  
+ 
+Thing: Car Wash   
+    5 properties:
+        - number of vacuum cleaners
+        - number of eco-friendly cleaning supplies
+        - stores the amount of water used per week.
+        - stores amount of profit made per week
+        - number of cars serviced per day
+    3 things it can do:
+        - wash and wax car
+        - charge customer
+        - detail the car interior
+
+    Notice that I did not use "has a vacuum cleaner" or "Has eco-friendly cleaning supplies" as one of the properties
+    
+    Writing 'has a ___" checks whether or not your object **has the ability to do something**.
+    Instead, I wrote "number of vacuum cleaners" and "number of eco-friendly cleaning supplies".  
+    
+    These are specific objects or amounts. 
+    
+    In C++ terms, this means to I want you to avoid using 'bool' (has a) as a member variable type.
+    Instead, prefer the other primitive types.
+ */
+
+/*
+Thing 1) multi-head echo machines
+5 properties:
+    1) number of playback head 
+    2) head spacing  
+    3) tape saturation (input gain preamp)  
+    4) spring reverb mix
+    5) output level
+3 things it can do:
+    1) distort signal
+    2) repeat sound source
+    3) play tape slower or faster
+ */
+
+/*
+Thing 2) solid state guitar amp
+5 properties:
+    1) number of speakers
+    2) speaker size 
+    3) master level
+    4) number of amp modeling presets
+    5) fx level
+3 things it can do:
+    1) digital signal processing
+    2) imitate tube amp  
+    3) amplifie sound 
+ */
+
+/*
+Thing 3) Sampling Pad
+5 properties:
+    1) number of pads
+    2) set click tempo bpm
```

Comment:
`set` is an action.  you have defined an action instead of a property.
- [ ] revise

Reply:
I fixed the issues